scrapi/tasks.py

```diff
@@ -35,7 +51,7 @@ def run_harvester(harvester_name, start_date=None, end_date=None):
     (start_harvest | normalization).apply_async()
 
 
-@app.task(default_retry_delay=300, max_retries=5)
+@app.task
```

Comment:
Retrying was getting a bit complicated what with those tasks themselves using chains and other things - decided to move most of the retry attempts to another future PR